pkg/kubelet/cm/pod_container_manager_linux.go

```diff
@@ -299,7 +299,7 @@ func (m *podContainerManagerNoop) EnsureExists(_ *v1.Pod) error {
 }
 
 func (m *podContainerManagerNoop) GetPodContainerName(_ *v1.Pod) (CgroupName, string) {
-	return m.cgroupRoot, m.cgroupRoot.ToCgroupfs()
+	return m.cgroupRoot, ""
```

Comment:
i think it's valid to move away from setting ToCgroupfs() for the systemd case.
and as far as i can see the `""` parent, would not get any special treatment in the case of the docker service.

but there could be hidden implications here.